plugins/modules/ovirt_vm_os_info.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_vm_os_info
+short_description: Retrieve information about all supported oVirt/RHV operating systems
+version_added: "1.0.1"
+author:
+- "Martin Necas (@mnecas)"
+description:
+    - "Retrieve information all supported oVirt/RHV operating systems."
+notes:
+    - "This module returns a variable C(ovirt_vm_os), which
+       contains a list of operating systems. You need to register the result with
+       the I(register) keyword to use it."
+options:
+    max:
```

Comment:
I removed this in my version as I couldn't think of a reason to limit the number of returned results?

Reply:
Good point I just use it everywhere.